ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -496,6 +501,11 @@ ORK_CLASS_AVAILABLE
  */
 @property (readonly) NSNumberFormatter *numberFormatter;
 
+/**
```

Comment:
minor nit - remove (read-only) from comment

Reply:
This wasn't a part of the PR. AFAICS it's meant to be read-only so that subclasses override it rather than assign to a property managed by ORKAnswerFormat itself.

Reply:
I was referring to the comment that is part of the PR for hideSelectedValue - "A Boolean value indicating whether the selected value should be hidden. (read-only)"